src/components/ContactsList/ContactsHeader.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const ContactsFilter = () => <div>ContactsFilter</div>;
+
+const ContactsIntentLink = () => <div>ContactsIntentLink</div>;
+
+const ContactsCreationButton = () => <div>ContactsCreationButton</div>;
+
+const ContactsHeader = () => (
+  <div style={{ border: "1px dashed silver", margin: "12px 0" }}>
```

Comment:
I don't think we should have bits of css in js if were using classes everywhere else.

Reply:
Those CSS rules are temporary rules that do not code wanted feature.